python/llm/src/ipex_llm/transformers/low_bit_linear.py

```diff
@@ -764,6 +764,7 @@ def __init__(self, input_features, output_features, bias=True,
         # weigh_type = 3 means weight has been transposed by esimd method
         self.weight_type = 1
         self.optimize_lm_head = optimize_lm_head
+        self.disable_fp16_opt = False
```

Comment:
I  wonder where did you set it to `True` for your use case ?

Reply:
On the AWQ side.